java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -1098,6 +1098,16 @@ void testStringManipulationWithNulls() {
     }
   }
 
+  @Test
+  void testInterleaveColumns() {
+    try (ColumnVector a = ColumnVector.fromInts(1,2,3,4,5);
+         ColumnVector b = ColumnVector.fromInts(6,7,8,9,10);
+         ColumnVector c = ColumnVector.fromInts(1,6,2,7,3,8,4,9,5,10);
+         ColumnVector d = ColumnVector.interleaveColumns(new ColumnVector[] { a, b })) {
+      assertColumnsAreEqual(c, d);
+    }
+  }
+
```

Comment:
Can we have some failure test cases?  Like what happens if I don't put in the same type of column.  What happens if I don't have the same number of rows, I know if you do the table API that is not possible? etc.

Reply:
I've added some failure test cases.